shell/platform/darwin/ios/framework/Source/FlutterAppDelegateTest.mm

```diff
@@ -80,8 +80,10 @@ - (void)testLaunchUrlWithDeepLinkingNotSet {
       [self.appDelegate application:[UIApplication sharedApplication]
                             openURL:[NSURL URLWithString:@"http://myApp/custom/route?query=test"]
                             options:@{}];
-  XCTAssertFalse(result);
-  OCMReject([self.mockNavigationChannel invokeMethod:OCMOCK_ANY arguments:OCMOCK_ANY]);
+  XCTAssertTrue(result);
```

Comment:
Can you have a test to verify set the flag to `No` will block deeplink?

Reply:
The test already exists in L89